pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -199,38 +200,38 @@ Here is a sample configuration file for the service:
 
 # Installation
 
-Now that you successfully created the config file (reffered to as `package-registry-options.json` from here on), you are ready to install the package registry. This can accomplished by following :
+Now that you have successfully created the config file (referred to as `package-registry-options.json` from here on), you are ready to install the package registry. This can accomplished by following :
 
 ```bash
 dcos package install package-registry --options=package-registry-options.json
 ```
 
-This would launch the marathon app for package-registry. This usually takes a couple of minutes. As soon as package-registry is healthy, you can add it as one of the package repositories in DC/OS. This can be done by:
+This would launch the Marathon app for `package-registry`. This usually takes a couple of minutes. As soon as `package-registry` is healthy, you can add it as one of the package repositories in DC/OS. This can be done by:
 
 ```bash
 # Change the repo name and URL if you need to customize
 dcos package repo add --index=0 "Registry" https://dcos-registry.marathon.l4lb.thisdcos.directory
 ```
 
-If you get errors in executing above command, wait for couple minutes (to account for the latency in package registry being healthy and its DNS entry being propogated to all master nodes) and try again.
+If you get errors in executing above command, wait for a couple of minutes (to account for the latency in `package-registry` being healthy and its DNS entry being propagated to all master nodes) and try again.
 
 # Using package registry
 
-After package registry is installed, you can start adding packages to it. There are two parts to this :
+After `package-registry` is installed, you can start adding packages to it. There are two parts to this :
```

Comment:
Rewrite `There are two parts to this :` with  `The two step process to use the package registry is as follows:`

Reply:
Done.